<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageCalibration">
       <layout class="QVBoxLayout" name="verticalLayout" stretch="3,1,1,12">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>内参矩阵</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>左摄像头</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>右摄像头</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>畸变系数</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>左摄像头</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>右摄像头</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,9,1,1,1,1,1,5,3">
           <item>
            <widget class="QPushButton" name="adddataBut">
             <property name="text">
              <string>add data</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="deletedataBut">
             <property name="text">
              <string>delete data</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>from</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>to</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_2"/>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="refreshBut">
             <property name="text">
              <string>refresh result</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pagePolar">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">background-image: url(:/image/welcome);
font: 48pt &quot;Agency FB&quot;;</string>
          </property>
          <property name="text">
           <string>                      Welcome !</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_file"/>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuCalibration">
    <property name="title">
     <string>Calibration</string>
    </property>
    <addaction name="actionTake_Photoes"/>
    <addaction name="actionSigalCamera"/>
    <addaction name="actionDual"/>
   </widget>
   <widget class="QMenu" name="menuPolar">
    <property name="title">
     <string>Polar Correction</string>
    </property>
    <addaction name="actionPolar_Correction"/>
   </widget>
   <widget class="QMenu" name="menu3d_recongnition">
    <property name="title">
     <string>3d reconstruction</string>
    </property>
    <addaction name="action3D_reconstruction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCalibration"/>
   <addaction name="menuPolar"/>
   <addaction name="menu3d_recongnition"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_file">
   <property name="text">
    <string>open file</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>save </string>
   </property>
  </action>
  <action name="actionDual">
   <property name="text">
    <string>Binocular</string>
   </property>
  </action>
  <action name="actionTake_Photoes">
   <property name="text">
    <string>Take Photoes</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>new</string>
   </property>
  </action>
  <action name="actionSigalCamera">
   <property name="text">
    <string>Sigal Camera</string>
   </property>
  </action>
  <action name="actionPolar_Correction">
   <property name="text">
    <string>Polar Correction</string>
   </property>
  </action>
  <action name="action3D_reconstruction">
   <property name="text">
    <string>3D reconstruction</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
